trulens_eval/trulens_eval/tests/groundedness_smoke_tests.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<p>Can you add a sentence on why SummEval is suitable for this evaluation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yes. Added description of the dataset on the top of the notebook.